server/src/test/proto/spine/test/event/project.proto

```diff
@@ -61,4 +61,5 @@ message Task {
     TaskId task_id = 1;
     string title = 2;
     string description = 3;
+    bool done = 4 [(required) = true];
```

Comment:
Let's call that `completed` or `released`. A project is not a task; it can hardly be "done".

Reply:
In fact, this is a task.